.github/workflows/ci-test-limited.yml

```
@@ -48,7 +48,13 @@ jobs:
         ports:
           # Opens tcp port 6379 on the host and service container
           - 6379:6379
+      postgres:
+        if: github.base_ref == 'pg' || github.ref_name == 'pg'
```

actionlint (1.7.4)

unexpected key "if" for "services" section. expected one of "credentials", "env", "image", "options", "ports", "volumes"

(syntax-check)